content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -218,3 +224,89 @@ applications:
+
+Out of the box, each binder middleware configuration file defines attributes to enable metrics and their supporting properties. You can find settings in: `src/kubernetes/server/server-config.yaml`. The main point of interest is the following configuration section:
+
+```yaml
+applicationProperties:
+  task:
+    management:
+      metrics:
+        export:
+          prometheus:
+            enabled: true
+            rsocket:
+              enabled: true
+              host: prometheus-proxy
+              port: 7001
+grafana-info:
+  url: 'https://grafana:3000'
+```
+
+In this configuration, Prometheus metrics are enabled and configured to send to a prometheus rsocket proxy.
+
+With Prometheus, Grafana, Spring Cloud Data Flow, and any other services as defined in the [Getting Started - Kubernetes](%currentPath%/installation/kubernetes) section up and running, you are ready to collect metrics.
+
+<!--IMPORTANT-->
+
+The address used to access the Grafana UI depends on the Kubernetes platform the system is deployed to. If you are using (for example) GKE, the load balancer address would be used. If using Minikube (which does not provide a load balancer implementation), the IP of the Minikube (along with an assigned port) is used. In the following examples, for simplicity, we use Minikube.
+
+<!--END_IMPORTANT-->
+
+To obtain the URL of the Grafana UI when it is deployed to Minikube, run the following command:
+
+```bash
+$ minikube service --url grafana
+http://192.168.99.100:31595
+```
+
+In the preceding example, you can reach the Grafana dashboard at http://192.168.99.100:31595. The default credentials are as follows:
+
+- User name: admin
+- Password: password
+
+The Grafana instance is pre-provisioned with two dashboards:
+
+- Tasks: http://192.168.99.100:31595/d/scdf-tasks/tasks?refresh=10s
+
+You can collect metrics on a per-task, per-batch basis or apply metrics collection to all deployed applications globally.
+
+To deploy a custom Task application (`task-demo-metrics`) and define two tasks (`task1` and `task2`):
+
+```bash
+dataflow:>app register --name myTask --type task --uri docker://tzolov/task-demo-metrics:latest
+
+dataflow:>task create --name task1 --definition "myTask"
+dataflow:>task create --name task2 --definition "myTask"
+```
+
+Launch the tasks several times:
+
+```bash
+dataflow:>task launch --name task1
+dataflow:>task launch --name task2
+```
+
+To obtain the SCDF URL. When it is deployed to Minikube, run the following command:
+
+```bash
+16:08 $ minikube service --url scdf-server
```

Comment:
Remove "16:08" and `$ symbol from everywhere. They get copied from the clipboard option otherwise.